### ISSUE TITLE
Use several meta-model in FamixMMUMLDocumentor with different color

### ISSUE THREAD

**Issue by badetitou** [moosetechnology/Famix#321]
Currently, the `FamixMMUMLDocumentor` allows generating the UML of one meta-model.
Entities from sub-meta-models are represented with _external entities_.
This is nice :-) ... but

Sometimes I want to represent a model and its sub-meta-models (maybe not all the sub-meta-models... but at least some).

Thus, is it possible to include several meta-models.
Something like:

```st
FamixMMUMLDocumentor new
 beWithStub;
 models: { CCModel . CCEModel };
 generatePlantUMLModel.
```
Even better, associate the model with a color

```st
FamixMMUMLDocumentor new
 beWithStub;
 withModel: CCModel andColor: Color blue;
 withModel: CCModel andColor: Color green;
 generatePlantUMLModel.
```

**Comment by enwiro** [comment]
It's true that this would be a really interesting addition. 

From memory, it would not be too difficult to implement, since in plantUML, entities are designated by their name. However, this could be a problem if there are two entities with the same name in the selected metamodels. It would be necessary to add their prefix.

Concerning the color, think that the simplest would be to generate a suffix in the plantUML code that defines the color.
One of the concerns I see with leaving the choice of colors to the user, is that they must match those available in the plantUML code.

In short, I'm following the FASTCore migration in traits for the moment. But as soon as it will be finished, I'll have a look at it ^^
